src/Wallabag/CoreBundle/Form/DataTransformer/StringToListTransformer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Wallabag\CoreBundle\Form\DataTransformer;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+
+class StringToListTransformer implements DataTransformerInterface
```

Comment:
Could you add a description about what this class is supposed to do and where it is (for now) used ?